core/services/flux_monitor.go

```diff
@@ -155,14 +159,17 @@ func (fm *concreteFluxMonitor) addAction(ctx context.Context, connected bool, jo
 	}
 	validCheckers := []DeviationChecker{}
 	for _, initr := range job.InitiatorsFor(models.InitiatorFluxMonitor) {
+		logger.Debugw(
+			fmt.Sprintf("adding job %s initr %d to flux monitor", job.ID.String(), initr.ID),
```

Comment:
I prefer to try and keep these lines very short, and have the interesting data kept in keys, since it's easier to parse out and read.

I've shown below an example of a good line and a line that's too long:
![Screen Shot 2019-12-26 at 1 10 10 PM](https://user-images.githubusercontent.com/344071/71489099-2f206b00-27e1-11ea-83d3-19755479ac6a.png)

Will this example fit? If not I suggest something like `"Added job to flux monitor", "jobId", ...`